modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -882,9 +903,14 @@ private ObjectRelationship _addObjectRelationship(
 			long userId, long objectDefinitionId1, long objectDefinitionId2,
 			long parameterObjectFieldId, String deletionType,
 			Map<Locale, String> labelMap, String name, boolean reverse,
-			String type)
+			boolean system, String type)
 		throws PortalException {
 
+		if (system && !SystemUtil.allowManageSystemEntities()) {
+			throw new ObjectRelationshipSystemException(
+				"Only allowed bundles can create system relationships");
```

Comment:
I think this word is missing: "Only allowed bundles can create system **object** relationships"